From 1d5c7ff88bb88a459eabe4f0245e6b92d1e40c31 Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Fri, 20 Mar 2026 14:15:57 -0300
Subject: [PATCH] csm-observability: change CSIs secret/configs name

This patch consists of changing the secret and config names used for
all dell CSIs.
It's needed because the names of the CSIs are created based on
the default release name and it were used hard-coded here.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
---
 .../templates/karavi-metrics-powerflex.yaml          | 12 ++++++------
 .../templates/karavi-metrics-powermax.yaml           | 10 +++++-----
 .../templates/karavi-metrics-powerscale.yaml         | 12 ++++++------
 .../templates/karavi-metrics-powerstore.yaml         |  2 +-
 4 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml b/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
index 5fce9cf..6db550a 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerflex.yaml
@@ -85,12 +85,12 @@ spec:
           - name: ACCESS_TOKEN
             valueFrom:
               secretKeyRef:
-                name: proxy-authz-tokens
+                name: csi-powerflex-proxy-authz-tokens
                 key: access
           - name: REFRESH_TOKEN
             valueFrom:
               secretKeyRef:
-                name: proxy-authz-tokens
+                name: csi-powerflex-proxy-authz-tokens
                 key: refresh
         volumeMounts:
           {{- if $authorizationConfigSecret }}
@@ -109,7 +109,7 @@ spec:
       volumes:
       - name: vxflexos-config
         secret:
-          secretName: vxflexos-config
+          secretName: csi-powerflex-config
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
@@ -124,14 +124,14 @@ spec:
       {{- if $authorizationConfigSecret }}
       - name: karavi-authorization-config
         secret:
-          secretName: karavi-authorization-config
+          secretName: csi-powerflex-authorization-config
       {{- end }}
       - name: proxy-server-root-certificate
         secret:
-          secretName: proxy-server-root-certificate
+          secretName: csi-powerflex-proxy-server-root-certificate
       - name: vxflexos-config-params
         configMap:
-          name: vxflexos-config-params
+          name: csi-powerflex-config-params
       {{ end }}
       {{ end }}
       restartPolicy: Always
diff --git a/charts/karavi-observability/templates/karavi-metrics-powermax.yaml b/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
index bae9c27..89cf8da 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powermax.yaml
@@ -99,12 +99,12 @@ spec:
           - name: ACCESS_TOKEN
             valueFrom:
               secretKeyRef:
-                name: powermax-proxy-authz-tokens
+                name: csi-powermax-proxy-authz-tokens
                 key: access
           - name: REFRESH_TOKEN
             valueFrom:
               secretKeyRef:
-                name: powermax-proxy-authz-tokens
+                name: csi-powermax-proxy-authz-tokens
                 key: refresh
         volumeMounts:
           {{- if $authorizationConfigSecret }}
@@ -146,14 +146,14 @@ spec:
       {{- if $authorizationConfigSecret }}
       - name: karavi-authorization-config
         secret:
-          secretName: powermax-karavi-authorization-config
+          secretName: csi-powermax-authorization-config
       {{- end }}
       - name: proxy-server-root-certificate
         secret:
-          secretName: powermax-proxy-server-root-certificate
+          secretName: csi-powermax-proxy-server-root-certificate
       - name: powermax-config-params
         configMap:
-          name: powermax-config-params
+          name: csi-powermax-config-params
       {{ end }}
       {{ end }}
       restartPolicy: Always
diff --git a/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml b/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
index 370e15f..f71e51d 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerscale.yaml
@@ -85,12 +85,12 @@ spec:
           - name: ACCESS_TOKEN
             valueFrom:
               secretKeyRef:
-                name: isilon-proxy-authz-tokens
+                name: csi-powerscale-proxy-authz-tokens
                 key: access
           - name: REFRESH_TOKEN
             valueFrom:
               secretKeyRef:
-                name: isilon-proxy-authz-tokens
+                name: csi-powerscale-proxy-authz-tokens
                 key: refresh
         volumeMounts:
           {{- if $authorizationConfigSecret }}
@@ -109,7 +109,7 @@ spec:
       volumes:
       - name: isilon-creds
         secret:
-          secretName: isilon-creds
+          secretName: csi-powerscale-creds
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
@@ -124,14 +124,14 @@ spec:
       {{- if $authorizationConfigSecret }}
       - name: karavi-authorization-config
         secret:
-          secretName: isilon-karavi-authorization-config
+          secretName: csi-powerscale-authorization-config
       {{- end }}
       - name: proxy-server-root-certificate
         secret:
-          secretName: isilon-proxy-server-root-certificate
+          secretName: csi-powerscale-proxy-server-root-certificate
       - name: isilon-config-params
         configMap:
-          name: isilon-config-params
+          name: csi-powerscale-config-params
       {{ end }}
       {{ end }}
       restartPolicy: Always
diff --git a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
index bfb90e6..7637f3e 100644
--- a/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
+++ b/charts/karavi-observability/templates/karavi-metrics-powerstore.yaml
@@ -115,7 +115,7 @@ spec:
               path: localhost.key
       - name: powerstore-config
         secret:
-          secretName: powerstore-config
+          secretName: csi-powerstore-config
       - name: tls-secret
         secret:
           secretName: otel-collector-tls
-- 
2.34.1

